sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1474,6 +1474,13 @@ object SQLConf {
     .checkValues((1 to 9).toSet + Deflater.DEFAULT_COMPRESSION)
     .createWithDefault(Deflater.DEFAULT_COMPRESSION)
 
+  val LEGACY_REPLACE_DATABRICKS_SPARK_AVRO_ENABLED =
+    buildConf("spark.sql.legacy.replaceDatabricksSparkAvro.enabled")
+      .doc("If it is set to true, the data source provider com.databricks.spark.avro is mapped " +
+        "to the built-in Avro data source module for backward compatibility.")
```

Comment:
do we want to give more details here about being for hive table provider compatibility?

I think it would also be nice if we put more details about the compatibility in the avro doc, either in this pr or in the other doc one.

Reply:
Thanks, I will do it in https://github.com/apache/spark/pull/22121